src/polly/src/event_manager.rs

```diff
@@ -246,354 +136,56 @@ impl EventManager {
         Ok(())
     }
 
-    // Dispatch an epoll event set for a handler.
-    #[inline(always)]
-    fn dispatch_event(
-        &mut self,
-        source: Pollable,
-        evset: epoll::EventType,
-        wrapped_handler: WrappedHandler,
-    ) -> Result<()> {
-        let mut all_ops = Vec::new();
-
-        let mut handler = wrapped_handler.lock().expect("Handler lock is poisoned");
-
-        // If an error occurs on a fd then only dispatch the error callback,
-        // ignoring other flags.
-        if evset.contains(epoll::EventType::ERROR) {
-            all_ops.append(&mut handler.handle_error(source));
+    /// Update an event handler pollables and event sets.
+    /// Use the PollableOpBuilder to build the vector of PollableOps.
```

Comment:
This seems a bit outdated.

Reply:
Comments are updated in the last PR, sorry for the confusion.